integration_test/mssql/test_helper.exs

```diff
@@ -0,0 +1,130 @@
+Logger.configure(level: :info)
+
+ExUnit.start(
+  exclude: [
+    :aggregate_filters,
+    :subquery_aggregates,
+    :array_type,
+    :modify_foreign_key_on_update,
+    :modify_foreign_key_on_delete,
+    :with_conflict_target,
+    :without_conflict_target,
+    :upsert_all,
+    :upsert,
+    :uses_msec,
+    :insert_cell_wise_defaults,
+    :select_not,
+    :coalesce_type,
+    :assigns_id_type,
+    :map_type_schemaless,
+    :map_boolean_in_expression,
+    :text_compare,
+    :decimal_type_cast,
+    :union_with_literals,
+    :db_engine_can_maintain_precision,
+    :inline_order_by,
+    :uses_usec,
+    # running destruction of PK columns requires that constraint is dropped first
+    :alter_primary_key,
+    :modify_column_with_from,
+    :on_replace_nulify,
+    :on_replace_update,
+    :unique_constraint_conflict,
+    :pk_insert,
+    :transaction_checkout_raises,
+    :transaction_multi_repo_calls,
+    # requires transaction isolation level to be set to ON and transaction isolation to :snapshot
+    :transaction_not_shared
+  ]
```

Comment:
I liked a lot that you have added comments to some of them. How feasible would be to add comments to all of them? Or at least group them accordingly? I will do the same for the other adapters. :)

Reply:
NP, I will add comments